app.py

```diff
@@ -204,6 +204,20 @@ def process_liquidity(main_chart_data: pandas.DataFrame, collateral_token: str,
 
     return main_chart_data, collateral_token_price
 
+
+def create_accumulated_debt(data):
```

Comment:
Please, add typing for parameters of your function and return type

1. It will help better to understand the data structure of this function and it's really useful while debugging